src/main/java/org/betonquest/betonquest/modules/updater/source/implementations/GitHubReleaseSource.java

```diff
@@ -16,13 +16,18 @@
  */
 public class GitHubReleaseSource extends UpdateSource implements ReleaseUpdateSource {
 
+    /**
+     * The basic url for the api of GitHub.
+     */
+    public static final String RELEASES_URL = "/releases";
     /**
      * The apiUrl to the GitHub releases API target.
```

Comment:
For me, a URL is an address I can enter in the browser to be redirected to a website. Accordingly, this attachment is for me more like the page path to the api of github.
I suggest the following:
"The page path to the GitHub releases API target".